File: src/main/java/dev/arbor/gtnn/data/GTNNTagPrefix.java
package dev.arbor.gtnn.data;

import com.gregtechceu.gtceu.api.data.tag.TagPrefix;
import dev.arbor.gtnn.GTNNIntegration;
import dev.arbor.gtnn.data.misc.AdAstraAddon;

public class GTNNTagPrefix {
    public static TagPrefix oreMoonStone;
    public static TagPrefix oreVenusStone;
    public static TagPrefix oreMarsStone;
    public static TagPrefix oreMercuryStone;
    public static TagPrefix oreGlacioStone;
    public static TagPrefix oreBlackStone;
    public static TagPrefix oreSoulSoil;

    public static void init() {
        if (GTNNIntegration.INSTANCE.isAdAstraLoaded()) AdAstraAddon.init();
    }
}

File: src/main/java/dev/arbor/gtnn/data/GTNNTagPrefix.kt
package dev.arbor.gtnn.data

import com.gregtechceu.gtceu.api.data.tag.TagPrefix
import dev.arbor.gtnn.GTNNIntegration.isAdAstraLoaded
import dev.arbor.gtnn.data.misc.AdAstraAddon

object GTNNTagPrefix {
    var oreMoonStone: TagPrefix? = null
    var oreVenusStone: TagPrefix? = null
    var oreMarsStone: TagPrefix? = null
    var oreMercuryStone: TagPrefix? = null
    var oreGlacioStone: TagPrefix? = null
    var oreBlackStone: TagPrefix? = null
    var oreSoulSoil: TagPrefix? = null

    fun init() {
        if (isAdAstraLoaded()) AdAstraAddon.init()
    }
}
